docs/plugins.rst

```diff
@@ -0,0 +1,136 @@
+Editor Plug-ins
+=================
+
+Even though the CodeGrade filesystem works together with any editor by manually editing its special files,
+it is recommended to use it in combination with editor plugins.
+
+.. note:: The CodeGrade filesystem has to be installed and both the ``cgfs`` and the ``cgapi-consumer`` helper program must be available from the user's ``$PATH`` to successfully install and use the editor plugins.
+
+Atom Plug-in
```

Comment:
This should link to the plugins, not contain a copy of their documentation.

Reply:
Fixed!